### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by sergejparity**
This PR adds CODEOWNERS file to the repository.
It is needed to increase awareness of the developers about related code changes.
Currently it adds CI team related stuff. But feel free to propose additions or changes to it either within this PR or later.

### CODE REVIEW

**Review by alvicsam — APPROVED** [review]

File: CODEOWNERS
Comment (ascjones) [inline CODEOWNERS:18]: I would have liked to be added as a global owner too, as per https://github.com/paritytech/ink/pull/1030

**Review by ascjones — COMMENTED** [review]
Looks okay to me, but please wait for our approval before merging here. If we are being slow just ping us.

File: CODEOWNERS
Comment (sergejparity) [reply CODEOWNERS:18]: My bad. Fixed this [here](https://github.com/paritytech/cargo-contract/pull/373)

**Pull request merged**
